s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -110,6 +111,13 @@ func (t *TidbClient) GetBucket(ctx context.Context, bucketName string) (bucket *
 		err = handleDBError(err)
 		return
 	}
+	//get versioning for the bucket
+	versionOpts,versionErr := t.GetBucketVersioning(ctx, tmp.Name)
+	if versionErr != nil {
+		return
```

Comment:
1. versionErr will not be returned here.
2. Add an error log please.

Reply:
Sure will add

Reply:
Done